e2e/capl-cluster-flavors/kubeadm-full-capl-cluster/chainsaw-test.yaml

```diff
@@ -214,6 +214,56 @@ spec:
             check:
               ($error == null): true
 
+    # Test to check if disks are added to the etcd spec
+    - clusters:
+        kubeadm-full-cluster:
+          kubeconfig: ./kubeadm-full-cluster-kubeconfig.yaml
+      name: Testing to check if disks are added to the spec
+      try:
+        - script:
+            env:
+              - name: TARGET_API
+                value: api.linode.com
+              - name:  TARGET_API_VERSION
+                value: v4beta
+              - name: URI
+                value: linode/instances
+              - name: FILTER
+                value: (to_string({"tags":($cluster)}))
+            content: |
+              set -e
+              
+              RESPONSE=$(curl -s \
+                -H "Authorization: Bearer $LINODE_TOKEN" \
+                -H "Content-Type: application/json" \
+                "https://$TARGET_API/$TARGET_API_VERSION/$URI")
```

Comment:
Use the `FILTER` to find the correct Linode ID here. You can't query for all the linodes and pick the first one. There is no guarantee its the linode instance associated with your test.